ebin/rebar.app

```diff
@@ -52,7 +53,11 @@
                   compiler,
                   crypto,
                   syntax_tools,
-                  tools]},
+                  tools,
+                  diameter,
```

Comment:
It didn't exist, but what can break?

We could add it via a `dialyzer_extra_app` list in rebar.config as an alternative, to keep the .app file portable.